include/ur_api.h

```diff
@@ -298,6 +298,7 @@ typedef enum ur_device_init_flag_t {
     /// @endcond
 
 } ur_device_init_flag_t;
+#define UR_DEVICE_INIT_FLAGS_MASK 0xffffffe0
```

Comment:
I'd like to see some documentation about what these are for?

Reply:
👍